docs/plugins/milestone.md

```diff
@@ -0,0 +1,46 @@
+# milestone
+
+## 设计背景
+
+通过命令让机器人来处理指派milestone的事务，同时可以添加一些说明来明确指派原因。
```

Comment:
There needs to be a space between Chinese and English. Also this place can think more about the motivation rather than the function.